presidential-elections/utils/gpapproximation.py

```diff
@@ -18,7 +18,7 @@ def make_sum_zero_hh(N: int) -> np.ndarray:
     e_1[0] = 1
     a = np.ones(N)
     a /= np.sqrt(a @ a)
-    v = a + e_1
+    v = e_1 - a
```

Comment:
:grimacing:

Reply:
Ha ha yeah, checked with the Labs team. The GP was already summing to 0... but not along the time axis, which is what we want. This fixes it